index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+    <link rel="stylesheet" type="text/css"  href="style.css" >
+  <script src="index.js" defer></script>
+</head>
+<body>
+<!-- Trigger/Open The Modal -->
+<button id="myBtn">Instructions</button>
+
+<!-- The Modal -->
+<div id="modal">
```

Comment:
as we're going to be using another modal later I would give this a more semantic id, ie instructionModal